mlrun/db/httpdb.py

```diff
@@ -102,6 +102,7 @@ def __init__(self, base_url, user="", password="", token=""):
         self._wait_for_background_task_terminal_state_retry_interval = 3
         self._wait_for_project_deletion_interval = 3
         self.client_version = version.Version().get()["version"]
+        self.python_version = version.Version().get_python_version(as_str=True)
```

Comment:
in python, avoid having an argument changing the return type, this is anti pattern.
```suggestion
        self.python_version = str(version.Version().get_python_version())
```

and have the `get_python_version` returns an object that implements the `__str__` as you expected it to be returned when using with `as_str`.